src/loaders/compute_tools/tool_common.py

```diff
@@ -148,21 +165,28 @@ def _parse_args(self):
 
         # Required flag arguments
         required.add_argument(
-            f'--{loader_common_names.LOAD_VER_ARG_NAME}', required=True, type=str,
-            help=loader_common_names.LOAD_VER_DESCR
+            f'--{loader_common_names.KBASE_COLLECTION_ARG_NAME}', required=True, type=str,
+            help=loader_common_names.KBASE_COLLECTION_DESCR
         )
         required.add_argument(
-            '--source_data_dir', required=True, type=str,
-            help='Source data (e.g. genome files) directory. '
-                 + '(e.g. /global/cfs/cdirs/kbase/collections/sourcedata/GTDB/r207'
+            f'--{loader_common_names.SOURCE_VER_ARG_NAME}', required=True, type=str,
+            help="Version of the source data, which should match the source directory in the collectionssource. (e.g. 207, 214 for GTDB, 2023.06 for GROW/PMI)"
```

Comment:
line wrap, but that can be fixed in a later PR AFAIC

Reply:
👍  I will fix it in the upcoming PR.